src/Action.ts

```diff
@@ -13,6 +13,7 @@ export enum ActionTypes {
   SET_ENTITY = 'SET_ENTITY',
   DISPATCH = 'DISPATCH',
   CHANGE_MODEL = 'CHANGE_MODEL',
+  FLOW = 'FLOW',
```

Comment:
Comment from other PR

`API_LOCAL` should be `CODE` or `CALLBACK` (which can call an API) it's just left over from log ago so I wouldn't based the new type on it.

Are we making this new action specific to Flow forever or eventually it would become general enough to call any IFTTT service. Would prefer to make it extensible, and be something like `HTTP_REQUEST` and (which happens to only work for flow now)

Reply:
Good question. I'll bubble up this question to Swadheen to see how soon we expect to work on supporting general HTTP request. As far as I know, the PVA is not supporting making a general HTTP request at this moment.